src/extension.ts

```diff
@@ -37,38 +37,7 @@ export async function activate(extensionContext: ExtensionContext): Promise<void
   extensionContext.subscriptions.push(
 
     extensionApi.commands.registerCommand('bootc.vfkit', async container => {
-      let imageLocation = container.labels['bootc.build.image.location'];
-
-      // Check that vfkit is installed and error if it isn't before executing
-      const telemetryData: Record<string, any> = {};
-      telemetryData.imageType = container.labels['bootc.build.type'];
-      try {
-        await extensionApi.process.exec('vfkit', ['--version']);
-      } catch (error) {
-        await extensionApi.window.showErrorMessage(`Unable to launch ${imageLocation} with vfkit: ${error}`);
-        telemetryData.error = 'no-version';
-        telemetryLogger.logUsage('launchVfkit', telemetryData);
-        return;
-      }
-
-      // Check to see if imageLocation exists and error if it doesn't before executing 
-      if (!fs.existsSync(imageLocation)) {
-        await extensionApi.window.showErrorMessage(`Unable to launch ${imageLocation} with vfkit: ${imageLocation} does not exist`);
-        telemetryData.error = 'no-image';
-        telemetryLogger.logUsage('launchVfkit', telemetryData);
-        return;
-      }
-
-      // Check container.labels['bootc.build.type'] to see if it is ami or raw
-      // if it is not raw or ami, we cannot launch with vfkit
-      if (container.labels['bootc.build.type'] !== 'ami' && container.labels['bootc.build.type'] !== 'raw') {
-        await extensionApi.window.showErrorMessage(`Unable to launch ${imageLocation} with vfkit: ${container.labels['bootc.build.type']} is not supported`);
-        telemetryData.error = 'unsupported-type';
-        telemetryLogger.logUsage('launchVfkit', telemetryData);
-        return;
-      }
-
-      await launchVfkit(imageLocation, telemetryData);
+      launchVFKit(container);
```

Comment:
I think you should await this call or catch the execution

probably need to add PR check about linter/formatter to this repository as well

Reply:
Just added missing await for now, but likely needs better error handling in future.

Yes, big +1 for linter/formatter - I made this repo compliant last week but it already has a few issues again. I've fixed those in my next PR but needs to be part of PR check very soon.